src/functions/menus/getNavigationMenu.js

```diff
@@ -1,10 +1,10 @@
 export default function getNavigationMenu(data) {
   const socialMedia = data[data?.length - 1];
   const { children } = socialMedia;
-  const menuProps = data.slice(1, 5).map(({ label, path }) => {
+  const menuProps = data.slice(1, 5).map(({ label }) => {
     return {
       label: label.toUpperCase(),
-      href: path,
+      href: `/${label.toLowerCase().split(" ").join("-")}`,
```

Comment:
Why are we using `label` and not `path` and `WORDPRESS_MULTISITE_SUBDIRECTORY`?

Reply:
@kilemensi For this i picked the label because the path has the pesayetu appended to the url. Um, i am not sure about using WORDPRESS_MULTISITE_PREFIX...wouldn't it affect custom links?